doc/source/cluster/faq.rst

```diff
@@ -0,0 +1,32 @@
+===
+FAQ
+===
+
+These are some Frequently Asked Questions that we've seen pop up for using Ray clusters.
+If you still have questions after reading this FAQ,  please reach out on
+`our Discourse <https://discuss.ray.io/>`__!
+
+Do Ray clusters support multi-tenancy?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, you can run multiple :ref:`jobs <jobs-overview>` simultaneously in a Ray cluster
+but it's NOT recommended in production.
+Instead, you can reuse the same cluster for running multiple jobs one at a time.
```

Comment:
what does this sentence mean? it sounds like multi-tenancy

Reply:
What I want to say is that it's not recommended to run multiple jobs concurrently since we don't have good isolation but running them sequentially is fine and you save the cost of starting a new cluster for each job.

Reply:
Updated. lmk if it's more clear.